console/other-usefull-commands.rst

```diff
@@ -21,6 +21,27 @@ This comes in handy if you e.g. added a new state that you need to translate for
  Translation.create_if_not_exists( :locale => 'de-de', :source => "New", :target => "Neu", format: 'string', created_by_id: 1, updated_by_id: 1 )
 
 
+Translating Attributes
+~~~~~~~~~~~~~~~~~~~~~~
+
+By default Zammad will not translate Attributes. A small exception here is the state dropdown, which has translation enabled by default.
```

Comment:
"attributes" instead of "Attributes"

That statement is not true. The following attributes get translated:
```
[2] pry(main)> ObjectManager::Attribute.all.to_a.keep_if{ |a| a.data_option[:translate] }.map{ |a| a.name }
=> ["type", "state_id", "pending_time", "priority_id", "type_id", "internal", "vip", "shared", "domain_assignment", "follow_up_possible", "follow_up_assignment"]
```

We should state instead that it only affects custom attributes.

Reply:
Updated the wording (to hopefully meet it better) with commit https://github.com/zammad/zammad-documentation/pull/95/commits/60ec1c2406273be9bd302bdfb0ad13cbdeafc686